src/tasks/start/start-backend.ts

```diff
@@ -74,10 +76,14 @@ export async function startBackend(
     daoFactoryAddress
   )
   logBack(`DAO deployed: ${dao.address}`)
+  const _experimentalAppInstaller = AppInstaller(
+    { apmAddress, dao, ipfsGateway },
+    bre
+  )
 
   // Call postDao hook.
   if (hooks && hooks.postDao) {
-    await hooks.postDao({ dao }, bre)
+    await hooks.postDao({ dao, _experimentalAppInstaller }, bre)
```

Comment:
I think it makes sense to include the app installer in `postDao`, `preInit` and `postInit` so the user has the flexibility to choose.

Reply:
Agree!